cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,283 @@
+            raise ValueError(f"Wrong Type {type(element)}")
+        return element
+
+    def get_body_name(self, event: Union[Tag, NavigableString, None]) -> str:
+        """
+        Get the body name for an event
+
+        Parameter:
+        ----------------
+        event: Union[Tag, NavigableString, None]
+            All elements in the page that we want to scrape
+
+        Returns:
+        ----------------
+        str
+            The body name
+        """
+        log.info("start get body name")
+        event = self.remove_extra_type(event)
+        bodyTable = event.find_all("table")[1].find("table")
+        if "CITY COUNCIL" in bodyTable.text:
+            return "City Council"
+        else:
+            return bodyTable.find_all("span")[3].text.title()
+
+    def get_event_Minutes_Item(
+        self,
+        event: Union[Tag, NavigableString, None],
+    ) -> List[ingestion_models.EventMinutesItem]:
+        """
+        Parse the page and gather the event minute items
+
+        Parameter:
+        ----------------
+        event: Union[Tag, NavigableString, None]
+            All elements in the page that we want to scrape
+
+        Returns:
+        ----------------
+        List[ingestion_models.EventMinutesItem]
+            All the event minute items gathered from the event on the page
+        """
+        log.info("start get items")
+        event_minutes_items = []
+        all_items = self.remove_extra_type(event).find_all("td", {"class": "style4"})
+        for item in all_items:
+            name = ""
+
+            for i in item.stripped_strings:
+                name = name + " " + repr(i).replace("'", "")
+
+            if name is not None and name != "":
+                event_minutes_items.append(
+                    ingestion_models.EventMinutesItem(
+                        minutes_item=ingestion_models.MinutesItem(name.strip())
+                    )
+                )
+
+        return event_minutes_items
+
+    # Big Functions
+    def get_diff_yearid(self, time: datetime) -> str:
+        """
+        Get the events in different years as the events for different
+        years are stored in different tabs. Can get multiple events
+        across years.
+
+        Parameters:
+        ---------------
+        time: datetime
+            The date of the event we are trying to parse
+
+        Returns:
+        ---------------
+        str
+            The year id that can locate the year tab where the event is stored
+        """
+        year = str(time.year)
+        year_id = "city-council-" + year
+        return year_id
+
+    def get_date_mainlink(self, element: BeautifulSoup) -> str:
+        """
+        Find the main link for one event.
+
+        Parameters:
+        --------------
+        time: datetime
+            The date of one event
+
+        Returns:
+        --------------
+        str
+            The main link, make agenda and video url in other function
```

Comment:
"make agenda and video url in other function" seems to be irrelevant for this particular function. If anything, I _think_ that text blob should be in a different docstring section, e.g. [Notes](https://numpydoc.readthedocs.io/en/latest/format.html#notes).

Reply:
Done